other-docs/guides/upgrading/v16.md

```diff
@@ -0,0 +1,57 @@
+---
+order: 16
+---
+
+# Upgrading to v16
+
+_If you are migrating from WordPress to Altis, check out
+the [migrating guide here](../migrating-from-wordpress.md) first._
+
+To upgrade to Altis v16, edit your `composer.json` and change the version
+constraint for `altis/altis` and any local
+environment modules to `^16.0.0`.
+
+```json
+{
+  "require": {
+	"altis/altis": "^16.0.0"
+  },
+  "require-dev": {
+	"altis/local-server": "^16.0.0"
+  },
+  "config": {
+	"platform": {
+	  "php": "8.0"
+	}
+  }
+}
+```
+
+Once you have made these changes run `composer update` and then run
+the `wp altis migrate` command:
+
+```sh
+# For cloud environments
+wp altis migrate
+
+# For local server
+composer server cli -- altis migrate
+```
+
+## Breaking Changes
+
+### PHP 8.2 ###
+
+Altis v16 fully supports PHP 8.2 in both local and cloud environments. There are a number of [backward incompatible changes](https://www.php.net/manual/en/migration82.incompatible.php) in PHP 8.2 which should be taken into consideration and addressed.
```

Comment:
We don't fully support as yet. Mostly because WordPress doesn't claim to fully support. So we definitely need to use the phrasing from the last release which was recently changed to clarify. I think it needs to be something like "Altis v16 continues to support PHP 8.2 as experimental. We are not aware of any breaking changes. but there are a number of ..."